chapters/sampling-randomization-power.tex

```diff
@@ -90,7 +90,7 @@ \subsection{Reproducibility in random Stata processes}
 Basically, it has a really long ordered list of numbers with the property that
 knowing the previous one gives you precisely zero information about the next one.
 Stata uses one of these numbers every time it has a task that is non-deterministic.
-In ordinary use, it will cycle through these numbers starting from a fixed point
+In ordinary use, it will cycle through these numbers starting from a new point
```

Comment:
```suggestion
In ordinary use, it will cycle through these numbers starting from a fixed point
```

Weirdly, this is correct as is. https://www.stata.com/manuals/rsetseed.pdf

> The sequences the random-number functions produce are determined by the seed, which is just a number and which is set to 123456789 every time Stata is launched. This means that runiform() produces the same sequence each time you start Stata.

Reply:
Ok, interesting. I did not know that and that seems like a very bad idea. #stata I will make sure it gets reverted and then I will close the comment.